WalletWasabi.Gui/Tabs/WalletManager/HardwareWallets/ConnectHardwareWalletViewModel.cs

```diff
@@ -324,10 +324,7 @@ public async Task<KeyManager> LoadKeyManagerAsync()
 				else
 				{
 					Logger.LogInfo("Hardware wallet was not used previously on this computer. Creating a new wallet file.");
-
-					var prefix = selectedWallet.HardwareWalletInfo is null ? "HardwareWallet" : selectedWallet.HardwareWalletInfo.Model.ToString();
-
-					walletName = WalletManager.WalletDirectories.GetNextWalletName(prefix);
+					walletName = WalletManager.WalletDirectories.GetNextWalletName(selectedWallet.HardwareWalletInfo?.Model.FriendlyName() ?? HardwareWalletModels.Unknown.FriendlyName());
```

Comment:
This line is quite long. Consider adding back that `prefix` variable if that makes sense to you. 

It would be also nice to move that `Logger.LogInfo` statement below this line and log the actual wallet name so that it's easy to follow what Wasabi Wallet does.

Reply:
Good points. Did both.